README.md

```diff
@@ -2,15 +2,23 @@
 
 [![Build Status](https://travis-ci.org/deis/charts.svg?branch=master)](https://travis-ci.org/deis/charts)
 
-This repository contains Helm Charts for Deis, the open source PaaS company.
+This repository contains Helm Charts for Deis, the open source PaaS company. Deis (pronounced DAY-iss) Workflow is an open source Platform as a Service (PaaS) that adds a developer-friendly layer to any [Kubernetes][k8s-home] cluster, making it easy to deploy and manage applications on your own servers.
 
-For more general-purpose Helm Charts, visit the [Helm Chart repository](https://github.com/helm/charts). To learn more about Helm, visit the [Helm repository](https://github.com/helm/helm).
+For more information about Deis Workflow, please visit the main project page at https://github.com/deis/workflow. For more general-purpose Helm Charts, visit the [Helm Chart repository](https://github.com/helm/charts). To learn more about Helm, visit the [Helm repository](https://github.com/helm/helm).
 
-## Work in Progress
+## Beta Status
 
-![Deis Graphic](https://s3-us-west-2.amazonaws.com/get-deis/deis-graphic-small.png)
+Many Deis components are currently in beta status, and we welcome your input! If you have feedback one one of the charts herein, please [submit an issue][issues]. If you'd like to participate in development, please read the "Development" section below and [submit a pull request][prs].
 
-Workflow and Helm are changing quickly. Your feedback and participation are more than welcome, but be aware that this project is considered a work in progress.
+# Development
```

Comment:
Is this really supposed to be a top level heading?  Seems like it should be `##`?